corehq/apps/dump_reload/sql/filters.py

```diff
@@ -37,21 +37,16 @@ def get_filters(self, domain_name):
 
 
 class UsernameFilter(DomainFilter):
-    def __init__(self, user_ids=None):
-        self.user_ids = user_ids
+    def __init__(self, usernames=None):
```

Comment:
wont this break https://github.com/dimagi/commcare-hq/blob/cb9d1ebd9385082d4daf16a3e0c8e0c6fca3613d/corehq/apps/dump_reload/sql/dump.py#L102 ? I assume that that always passes IDs

Reply:
no it won't break it since that doesn't pass in anything